test/util.ts

```diff
@@ -0,0 +1,3 @@
+export function onWindows(): boolean {
```

Comment:
Can we just add this as an `const isWindows = process.platform === 'win32';` at the top of the file?  No need to have this separate for now, and this file would need a lic header if we did

Reply:
Sure! I was not sure if we needed this in more than one file when I did that. Changing.